### ISSUE TITLE
Wiki page for Hospital

### ISSUE THREAD

**Issue by JCharante** [baxter-cs/AndOre#201]
Parent Issue: https://github.com/baxter-cs/AndOre/issues/198

**Comment by JCharante** [comment]
Pull request (https://github.com/baxter-cs/AndOre/pull/204) created, waiting for code review and then once this is merged into unstable the issue will be closed.
